bin/restrax

```diff
@@ -0,0 +1,583 @@
+#!/usr/bin/env python
+"""
+Restrax: Rechunking live data
+=============================================
+How to use
+----------------
+    <activate conda environment>
+    restrax --production
+
+----------------
+
+For more info, see the documentation:
+https://straxen.readthedocs.io/en/latest/scripts.html
+"""
+__version__ = '0.1.0'
+
+import argparse
+import logging
+import os
+import immutabledict
+import socket
+import shutil
+import time
+import numpy as np
+import strax
+import straxen
+import threading
+import daqnt
+import fnmatch
+import typing as ty
+from straxen import daq_core
+from memory_profiler import memory_usage
+import glob
+from ast import literal_eval
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="XENONnT rechunking manager")
+    parser.add_argument('--production', action='store_true',
+                        help="Run restrax in production mode, otherwise run in a test mode.")
+    parser.add_argument('--ignore_checks', action='store_true',
+                        help="Do not use! Skip checks before changing documents, there be dragons.", )
+    parser.add_argument('--input_folder', type=str, default=daq_core.pre_folder,
+                        help="Where to read the data from, should only be used when testing", )
```

Comment:
I would prefer to keep it as it is to avoid cluttering the help with too many arguments that only are relevant when testing things. Is it okay if i leave it as it is?

Reply:
Ah I get it, fine by me!